package/kedro_viz/integrations/deployment/base_deployer.py

```diff
@@ -67,7 +67,7 @@ def _write_heap_injected_index(self, html_content):
             with open(temp_file_path, "w", encoding="utf-8") as temp_index_file:
                 temp_index_file.write(html_content)
 
-            self._fs.put(temp_file_path, f"{self._path}/")
+            self._fs.put(temp_file_path, f"{self._path}/", content_type="text/html")
```

Comment:
Does it works for all deployers?

Reply:
yes it works for AWS, GCP and local. AzureDeployer has its own `_write_heap_injected_index` method